doc/Makefile

```diff
@@ -141,7 +141,7 @@ install:
 	# --depth 1 is a speed optimization since we don't need the
 	# history prior to the last commit
 	# -b gh-pages fetches only the branch for the gh-pages
-	git clone -b gh-pages --single-branch --no-checkout --depth 1 https://github.com/glm-tools/pyglmnet _build/pyglmnet.github.io
+	git clone -b gh-pages --single-branch --no-checkout --depth 1 git@github.com:glm-tools/pyglmnet.git _build/pyglmnet.github.io
```

Comment:
while you're at it, can you also update L137 to point to `glm-tools`? Thanks!

Reply:
updated in another pull request
I accidentally deleted the fork
Sorry!